openpype/settings/defaults/project_settings/deadline.json

```diff
@@ -95,6 +70,33 @@
             "group": "",
             "department": "",
             "multiprocess": true
+        },
+        "ProcessSubmittedJobOnFarm": {
+            "enabled": true,
+            "deadline_department": "",
+            "deadline_pool": "",
+            "deadline_group": "",
+            "deadline_chunk_size": 1,
+            "deadline_priority": 40,
```

Comment:
This default value has been changed from `50` to `40`. Is that intended?

Reply:
Nope, fixed